docs/Misc/Troubleshooting.md

```diff
@@ -22,4 +25,4 @@ This is, admittedly, fairly unintuitive. This is subject to change in future upd
 ## A mod I used to be able to play with now refuses to be enabled
 It's likely that this mod has some sort of error in its XML that is making it unable to function properly; the game now disables mods with such errors to prevent obscure problems further down the road. If you're the develop of this mod, you may need to edit it to correct these errors. It's also entirely possible that you've encountered a bug with the game, so if you suspect that this is the case, let us know by [submitting a detailed report on our issue tracker](https://github.com/Regalis11/Barotrauma/issues)!
 
-If the error you see is "Failed to add the prefab X from MyMod: a prefab with the same identifier from OtherMod already exists; try overriding": This occurs when two prefabs with the same identifier are defined, and neither of them is marked to [override](../Intro/Overrides.md) the other. To solve this, you will either need to change the identifier of the mentioned prefab in your mod, or wrap it in `<override>...</override>` tags.
\ No newline at end of file
+If the error you see is "Failed to add the prefab X from MyMod: a prefab with the same identifier from OtherMod already exists; try overriding": This occurs when two prefabs with the same identifier are defined, and neither of them is marked to [override](../Intro/Overrides.md) the other. To solve this, you will either need to change the identifier of the mentioned prefab in your mod, or wrap it in `<override>...</override>` tags.

```

Comment:
I don't actually see a diff here and a separate diff-checker says [the lines are identical](https://www.diffchecker.com/uvAz6dFh). Might be a GitHub quirk.